docs/admin-manual/cm-configuration.rst

```diff
@@ -833,4 +940,819 @@ Running with multiple negotiators also means you need to be careful with the
+
+Absent ClassAds
+---------------
+
+:index:`absent ClassAds<single: absent ClassAds; pool management>`
+:index:`absent ClassAd` :index:`absent ClassAd<single: absent ClassAd; ClassAd>`
+
+By default, HTCondor assumes that slots are transient: the
+*condor_collector* will discard ClassAds older than :macro:`CLASSAD_LIFETIME`
+seconds. Its default configuration value is 15 minutes, and as such, the
+default value for :macro:`UPDATE_INTERVAL` will pass three times before
+HTCondor forgets about a resource. In some pools, especially those with
+dedicated resources, this approach may make it unnecessarily difficult to
+determine what the composition of the pool ought to be, in the sense of knowing
+which machines would be in the pool, if HTCondor were properly functioning on
+all of them.
+
+This assumption of transient machines can be modified by the use of absent
+ClassAds. When a slot ClassAd would otherwise expire, the *condor_collector*
+evaluates the configuration variable :macro:`ABSENT_REQUIREMENTS` against the
+machine ClassAd. If ``True``, the machine ClassAd will be saved in a persistent
+manner and be marked as absent; this causes the machine to appear in the output
+of ``condor_status -absent``. When the machine returns to the pool, its first
+update to the *condor_collector* will invalidate the absent machine ClassAd.
+
+Absent ClassAds, like offline ClassAds, are stored to disk to ensure that they
+are remembered, even across *condor_collector* crashes. The configuration
+variable :macro:`COLLECTOR_PERSISTENT_AD_LOG` defines the file in which the
+ClassAds are stored.
+Absent ClassAds are retained on disk as maintained by the *condor_collector*
+for a length of time in seconds defined by the configuration variable
+:macro:`ABSENT_EXPIRE_ADS_AFTER`. A value of 0 for this variable means that the
+ClassAds are never discarded, and the default value is thirty days.
+
+Absent ClassAds are only returned by the *condor_collector* and displayed when
+the **-absent** option to *condor_status* is specified, or when the absent
+machine ClassAd attribute is mentioned on the *condor_status* command line.
+This renders absent ClassAds invisible to the rest of the HTCondor
+infrastructure.
+
+A daemon may inform the *condor_collector* that the daemon's ClassAd should not
+expire, but should be removed right away; the daemon asks for its ClassAd to be
+invalidated. It may be useful to place an invalidated ClassAd in the absent
+state, instead of having it removed as an invalidated ClassAd. An example of a
+ClassAd that could benefit from being absent is a system with an
+uninterruptible power supply that shuts down cleanly but unexpectedly as a
+result of a power outage. To cause all invalidated ClassAds to become absent
+instead of invalidated, set :macro:`EXPIRE_INVALIDATED_ADS` to ``True``.
+Invalidated ClassAds will instead be treated as if they expired, including when
+evaluating :macro:`ABSENT_REQUIREMENTS`.
+
+GPUs
```

Comment:
Should this GPU section be in the cm-configuration and not the ep-configuration?

Reply:
I went back and forth on that...

Reply:
Decided to leave GPU section in the CM configuration page. If needed we can move this to the EP configuration page or add a link to this section